packages/components/src/Menu/Menu.test.tsx

```diff
@@ -74,244 +24,76 @@ const mockMatchMedia = (matches: string[]) =>
-  fireEvent.click(burger);
-
-  const pizza = screen.queryByText('Pizza');
-  expect(burger).not.toBeInTheDocument();
-  expect(pizza).not.toBeInTheDocument();
-});
-
-test('closes menu when trigger is clicked', () => {
-  render(
-    <Menu label="Choose">
-      <Menu.Item key="burger">Burger</Menu.Item>
-      <Menu.Item key="pizza">Pizza</Menu.Item>
-    </Menu>
-  );
-
-  const button = screen.getByText('Choose');
-  user.click(button);
-  user.click(button);
-
-  const burger = screen.queryByText('Burger');
-  const pizza = screen.queryByText('Pizza');
-
-  expect(burger).not.toBeInTheDocument();
-  expect(pizza).not.toBeInTheDocument();
-});
-
-test('closes menu when clicked outside', async () => {
-  render(
-    <>
-      <Button>outside</Button>
-
-      <Menu label="Choose">
-        <Menu.Item key="burger">Burger</Menu.Item>
-        <Menu.Item key="pizza">Pizza</Menu.Item>
-      </Menu>
-    </>
-  );
-
-  const button = screen.getByText('Choose');
-  await user.click(button);
-
-  const burger = screen.getByText('Burger');
-  const pizza = screen.queryByText('Pizza');
-
-  const outside = screen.getByText('outside');
-  await user.click(outside);
-
-  expect(burger).not.toBeInTheDocument();
-  expect(pizza).not.toBeInTheDocument();
-});
-
-test('return action item', async () => {
-  const spy = vi.fn();
-  render(
-    <Menu label="Choose" onAction={spy}>
-      <Menu.Item id="burger">Burger</Menu.Item>
-      <Menu.Item id="pizza">Pizza</Menu.Item>
-    </Menu>
-  );
-
-  const button = screen.getByText('Choose');
-  await user.click(button);
-
-  const burger = screen.getByText('Burger');
-  await user.click(burger);
-
-  expect(spy).toHaveBeenCalledWith('burger');
-  expect(spy).not.toHaveBeenCalledWith('pizza');
-});
-
-test('uses base classes from "Menu" in theme', () => {
-  render(
-    <Menu data-testid="menu" label="Choose">
-      <Menu.Item key="burger">Burger</Menu.Item>
-      <Menu.Item key="pizza">Pizza</Menu.Item>
-    </Menu>
-  );
-  const button = screen.getByText('Choose');
-  fireEvent.click(button);
-
-  const menu = screen.getByRole('menu');
-  expect(menu).toHaveClass(`bg-white`);
+  expect(gryffindor).not.toBeInTheDocument();
+  expect(hufflepuff).not.toBeInTheDocument();
+  expect(ravenclaw).not.toBeInTheDocument();
+  expect(slytherin).not.toBeInTheDocument();
 });
 
 test('supports "Menu" variant classnames from theme', () => {
-  render(
-    <Menu data-testid="menu" label="Choose" variant="one">
-      <Menu.Item key="burger">Burger</Menu.Item>
-      <Menu.Item key="pizza">Pizza</Menu.Item>
-    </Menu>
-  );
-  const button = screen.getByText('Choose');
+  render(<Basic data-testid="menu" variant="one" />);
+  const button = screen.getByText('Hogwarts Houses');
   fireEvent.click(button);
 
   const menu = screen.getByRole('menu');
-  const item = screen.getByText('Burger');
+  const item = screen.getByText('Gryffindor');
 
   expect(menu.className).toMatchInlineSnapshot(
```

Comment:
not sure if we should keep these tests, they will always break if we change something in the theme (e.g. when we make RUI the default)

I wonder if we can delete these tests alltogether (meaning the "is the variant/size applied" tests)

@marigold-ui/developer

Reply:
For me its ok to delete it, I can also do it in a seperate PR

Reply:
yeah lets talk about it in mob tomorrow?